moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -73,12 +73,20 @@ const useStyles = makeStyles((theme) => ({
     margin: "8px 0",
     padding: "8px",
   },
+  editIconConfirm: {
+    cursor: "pointer",
+    margin: ".25rem 0",
+    fontSize: "24px",
+  },
```

Comment:
I deleted a lot of "stale" styles in a previous PR, but then discovered that this one was passed through to a different component. And the other style was also not here but is also needed by a child component.

Reply:
Thanks for restoring these! This is probably a good argument for us to avoid passing the `classes` object as a prop in the future so it is easier to see what is used where. 

I'm guilty of this for sure, and I ran into it in the components code too recently.